sections/editing.include

```diff
@@ -1667,14 +1668,24 @@
   If specified, the value must be a single printable character:
   typically a string one Unicode code point in length.
   
-  <p class="warning">Character references such as <code>accesskey="&#65"</code> are not valid values, 
-  because they will cause no shortcut to be assigned in many browsers. 
-  However certain printable characters that may represent more than one Unicode code point, such as 
-  <code>accesskey="श्र"</code> are valid.</p>
+  <div class="warning">
+    Composed characters in
+    <a href="https://www.w3.org/International/questions/qa-html-css-normalization">Normalised Form D</a>,
+    such as <code>accesskey="&amp;#x006E;&amp;#x0303;"</code>
+    to assign <samp>ñ</samp> as a shortcut, are not valid values,
+    and in many browsers no shortcut will be assigned. Characters in
+    <a href="https://www.w3.org/International/questions/qa-html-css-normalization">Normalised Form C</a>
+    are valid values. Printable characters that may represent more than one Unicode code point, such as 
+    <code>accesskey="श्र"</code>, are valid values.
   
```

Comment:
This is odd to me. Indic sequences of characters are ok, but not latin ones(?)  Is it because something is normalising the shortcut values to precomposed (NFC) form before setting up the event?  
'are not valid values' suggests that the spec is outlawing them.  Did you mean to say that they 'are often not accepted by browsers' and/or 'may not match what the user's keyboard produces, since they are usually precomposed forms'?

Is there a definition of 'printable character' that can be pointed to?